internal/mapping.go

```diff
@@ -144,27 +144,46 @@ func GetSpannerCols(conv *Conv, srcTable string, srcCols []string) ([]string, er
 // (across the database). But in some source databases, such as PostgreSQL,
 // they only have to be unique for a table. Hence we must map each source
 // constraint name to a unique spanner constraint name.
-func GetSpannerKeyName(srcKeyName string, schemaForeignKeys map[string]bool) string {
-	if srcKeyName == "" {
+func ToSpannerForeignKey(srcId string, used map[string]bool) string {
+	if srcId == "" {
 		return ""
 	}
-	spKeyName, _ := FixName(srcKeyName)
-	if _, found := schemaForeignKeys[spKeyName]; found {
+	return getSpannerId(srcId, used)
+}
+
+// ToSpannerIndexKey maps source index key name to
```

Comment:
Can we clean up the comment a little.
- should use full sentences and punctuation e.g. add full stop after first sentence.
- format so its more readable e.g. join the couple of short lines of text together such as lines 156 and 157.
- be consistent about terminology e.g. index key vs index key name vs constraint name; maybe we should use index name everywhere (including the function name).

Reply:
sure, updated comment